packages/backend/src/studio-api-impl.ts

```diff
@@ -157,6 +157,34 @@ export class StudioApiImpl implements StudioAPI {
     return this.environmentManager.getEnvironmentsState();
   }
 
+  async requestRemoveEnvironment(recipeId: string): Promise<void> {
+    const recipe = this.catalogManager.getRecipeById(recipeId);
+    // Do not wait on the promise as the api would probably timeout before the user answer.
```

Comment:
Not sure the comment is relevant

Reply:
I think it is, to explain why we are not wait'ing for the Promise returned by showWarningMessage to be resolved, but are using `then` instead (or the request between front and back timeouts)